### PR TITLE
pythonPackages.pytest-dotenv: init at 0.5.2

### PR DESCRIPTION
**Pull request by cleeyv**
###### Motivation for this change

The pytest plugin pytest-dotenv is a dependency of the tests for LiberaForms which I packaged as a flake and module for Summer of Nix:  https://github.com/ngi-nix/liberaforms

###### Things done

I have tested that this package works by using it in the flake.nix for LiberaForms and successfully running all of the tests with `nix flake check`.

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/pytest-dotenv/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  checkInputs = [ pytest ];
+
+  meta = with lib; {
+    description = "A py.test plugin that parses environment files before running tests";
+    homepage = "https://github.com/quiqua/pytest-dotenv";
+    license = licenses.mit;
+    maintainers = with maintainers; [ cleeyv ];
+  };
+}
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pytest-dotenv/default.nix:24]: One new line to much.

File: pkgs/development/python-modules/pytest-dotenv/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    sha256 = "LcbDrG2HZMccbSgE6QLQ/4EPoZaS6V/hOK78mxqnNzI=";
+  };
+
+  buildInputs = [ pytest ];
+  propagatedBuildInputs = [ python-dotenv ];
+
+  checkInputs = [ pytest ];
+
+  meta = with lib; {
+    description = "A py.test plugin that parses environment files before running tests";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pytest-dotenv/default.nix:18]:
```suggestion
    description = "A pytest plugin that parses environment files before running tests";
```

File: pkgs/development/python-modules/pytest-dotenv/default.nix (hunk 1 above)
Comment (cleeyv) [reply pkgs/development/python-modules/pytest-dotenv/default.nix]: Thanks.

File: pkgs/development/python-modules/pytest-dotenv/default.nix (hunk 2 above)
Comment (cleeyv) [reply pkgs/development/python-modules/pytest-dotenv/default.nix]: Agreed.

**Pull request merged**